_sass/_variables.scss

```diff
@@ -19,6 +21,7 @@ $helvetica: Helvetica, Arial, sans-serif;
 $helveticaNeue: "Helvetica Neue", Helvetica, Arial, sans-serif;
 $georgia: Georgia, serif;
 $founderGroteskRegular: "FoundersGroteskRegular", Helvetica, Arial, sans-serif;
+$tiemposHeadlineRegular: "TiemposHeadlineRegular", Times New Roman, serif;
```

Comment:
@rx , I had included this font, and then didn't see where the Maxwell blog was using it, so took it out.